packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -197,7 +197,7 @@ export interface SdkModelType extends SdkTypeBase {
   details?: string;
   access: AccessFlags | undefined;
   usage: UsageFlags;
-  additionalProperties: boolean;
+  additionalProperties: SdkType | undefined;
```

Comment:
The docs define this as `additionalProperties?: SdkType;` which is slightly different, but I assume is identical?